packages/prism-theme-odyssey/src/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Prism Theme Odyssey</title>
+      <style>
+        main {
+          max-width: 960px;
+          margin: 48px auto;
+        }
+
+        section {
+          padding: 48px 0 !important;
+        }
+      </style>
+   </head>
+   <body>
+    <main>
+      <h1>Prism Theme Odyssey</h1>
+      <section>
+        <h2>HTML</h2>
+        <pre><code class="language-html">&lt;!DOCTYPE html&gt;
+  &lt;html lang="en"&gt;
+    &lt;head&gt;
+      &lt;meta charset="utf-8"&gt;
+      &lt;title&gt;Hello world&lt;/title&gt;
```

Comment:
Looks like we might have some cleanup here too.

Reply:
Moved to #730